src/portal/src/app/base/harbor-shell/harbor-shell.component.html

```diff
@@ -29,7 +29,7 @@
                             <clr-icon shape="users" clrVerticalNavIcon></clr-icon>
                             {{'SIDE_NAV.SYSTEM_MGMT.USER' | translate}}
                         </a>
-                        <a *ngIf='isLdapMode' clrVerticalNavLink routerLink="/harbor/groups" routerLinkActive="active">
+                        <a *ngIf='isLdapMode || isOidcMode' clrVerticalNavLink routerLink="/harbor/groups" routerLinkActive="active">
```

Comment:
why http mode does not have this?

Reply:
yes, it should have http mode, let me check why not.

Reply:
Oh I found that I made a wrong change in an add white list commit,I correct it.
commit 274ab7bcffaf48d5d70458a21cc2b18639787c53